docs/nddata/utils.rst

```diff
@@ -26,6 +26,8 @@ postage stamp cutout image from a 2D array.  If an optional
 
 First, let's simulate a single source on a 2D data array. If you would like to
 simulate many sources, see :ref:`bounding-boxes`.
+.. note:: The pair convention is inconsistent. The position argument is specified
+.. as (x,y), however the size argument is reverse (y,x)
```

Comment:
You would need to fix the RST formatting. I'll leave subpackage maintainer to review the actual content.